src/main/java/fridgy/model/ingredient/BaseIngredient.java

```diff
@@ -0,0 +1,95 @@
+package fridgy.model.ingredient;
+
+import java.util.Objects;
+
+import fridgy.commons.util.CollectionUtil;
+import fridgy.model.base.Eq;
+
+/**
+ * Represents a Base ingredient in the Inventory, to be used by Recipe.
```

Comment:
I don't think it's used in Inventory. Its purpose is to draw relation between ingredient in Inventory and Recipe so they have a basis for comparison. I think 

`Represents an basic ingredient used in Recipe` will suffice for the javadoc here.